group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorShard.java

```diff
@@ -262,6 +267,29 @@ public HeartbeatResponseData genericGroupHeartbeat(
         );
     }
 
+    public CoordinatorResult<DeleteGroupsResponseData.DeletableGroupResultCollection, Record> deleteGroups(
+            RequestContext context,
+            List<String> groupIds
```

Comment:
nit: Indentation. There are other cases in this PR. I won't mention them all.

Reply:
Fixed